src/coreclr/pal/src/exception/machexception.cpp

```diff
@@ -849,7 +849,7 @@ HijackFaultingThread(
     if (fIsStackOverflow)
     {
         // Allocate the minimal stack necessary for handling stack overflow
-        int stackOverflowStackSize = 7 * 4096;
+        int stackOverflowStackSize = 15 * 4096;
```

Comment:
@janvorli This looks like part of the fix for https://github.com/dotnet/runtime/issues/98477 that you have been working on.

Reply:
Yes, @mikem8361 has hit the issue, so I've told him he could make this part of the change if he wants to. I'll fix the stackoverflow detection in the presence of the macos issue.

Reply:
I wanted to get the couple of changes in so I can get my diagnostics repo .NET testing in soon.